test/integration/specialist_document_test.rb

```diff
@@ -2,10 +2,22 @@
 
 class SpecialistDocumentTest < ActionDispatch::IntegrationTest
   test "random but valid items do not error" do
-    setup_and_visit_random_content_item(document_type: 'aaib_report')
-    setup_and_visit_random_content_item(document_type: 'raib_report')
-    setup_and_visit_random_content_item(document_type: 'tax_tribunal_decision')
-    setup_and_visit_random_content_item(document_type: 'cma_case')
+    setup_and_visit_content_item('aaib-reports')
+    setup_and_visit_content_item('asylum-support-decision')
+    setup_and_visit_content_item('business-finance-support-scheme')
+    setup_and_visit_content_item('cma-cases')
+    setup_and_visit_content_item('countryside-stewardship-grants')
+    setup_and_visit_content_item('drug-safety-update')
+    setup_and_visit_content_item('employment-appeal-tribunal-decision')
+    setup_and_visit_content_item('employment-tribunal-decision')
+    setup_and_visit_content_item('european-structural-investment-funds')
+    setup_and_visit_content_item('international-development-funding')
+    setup_and_visit_content_item('maib-reports')
+    setup_and_visit_content_item('raib-reports')
+    setup_and_visit_content_item('residential-property-tribunal-decision')
```

Comment:
This line should probably be introduced by the other PR #708 which we'll rebase on top of this one I guess?

Reply:
Sure